core/src/test/scala/unit/kafka/server/ControllerApisTest.scala

```diff
@@ -732,6 +730,79 @@ class ControllerApisTest {
       controllerApis.createPartitions(request, false, _ => Set("foo", "bar")).get().asScala.toSet)
   }
 
+  @Test
+  def testElectLeadersAuthorization(): Unit = {
```

Comment:
Similar question: why not use `createControllerApis().handleElectLeaders` ?

Reply:
That would skip the exception handler in `ControllerApis.handle`.